### DISCUSSION TITLE
Multi-partition simultaneous retrieval

### DISCUSSION THREAD

**Discussion by gggdroa** [milvus-io/milvus#37666] [Q&A and General discussion]
Can multiple partitions be searched at the same time?

**Comment by yhmo** [comment]
Sure.

```
results = client.search(
    collection_name=collection_name,
    partition_names=["p1", "p2"],
    data=vectors,
    limit=10,
    search_params={
        "metric_type": "COSINE",
    },
    anns_field=vector_field_name,
)
```

**Comment by gggdroa** [comment]
ok  tks

---- Replied Message ----
| From | ***@***.***> |
| Date | 11/14/2024 15:22 |
| To | milvus-io/milvus ***@***.***> |
| Cc | gggdroa ***@***.***>,
Author ***@***.***> |
| Subject | Re: [milvus-io/milvus] Multi-partition simultaneous retrieval (Discussion #37666) |

Sure.

results = client.search(
    collection_name=collection_name,
    partition_names=["p1", "p2"],
    data=vectors,
    limit=10,
    search_params={
        "metric_type": "COSINE",
    },
    anns_field=vector_field_name,
)
